spark-extension/src/main/java/org/apache/spark/sql/auron/AuronConf.java

```diff
@@ -67,64 +68,138 @@ public enum AuronConf {
     /// always skip partial aggregate when triggered spilling
     PARTIAL_AGG_SKIPPING_SKIP_SPILL("spark.auron.partialAggSkipping.skipSpill", false),
 
-    // parquet enable page filtering
+    /// parquet enable page filtering
```

Comment:
Should we refer to Spark's ConfigEntry solution and write the comment of the configuration item to the doc field?

Reply:
> Should we refer to Spark's ConfigEntry solution and write the comment of the configuration item to the doc field?

i‘d like to use a more flexible solution for supporting other engines than Spark in the future.

Reply:
Maybe I'm not clear about it, I mean that we can maintain Spark-like ConfigEntry like Kyuubi and Celeborn.

https://github.com/apache/kyuubi/blob/master/kyuubi-common/src/main/scala/org/apache/kyuubi/config/ConfigEntry.scala#L20

https://github.com/apache/celeborn/blob/main/common/src/main/scala/org/apache/celeborn/common/internal/config/ConfigEntry.scala#L76